blackbox/docs/general/builtins/aggregation.rst

```diff
@@ -31,13 +31,48 @@ query result into one row, with one returned aggregation value.
 For a tabulated summary of aggregation functions, see
 :ref:`sql_dql_aggregation`.
 
+.. _aggregate-expressions:
+
+Aggregate Expressions
+=====================
+
+An aggregate expression represents the application of an aggregate function
+across rows selected by a query. An aggregate function reduces multiple
+input values to a single output value, for instance, the ``sum``, ``average``,
+or ``count`` of the input values.
+
+Synopsis
+--------
+
+::
+
+   aggregate_function ( * ) [ FILTER ( WHERE condition ) ]
+   aggregate_function ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE condition ) ]
+
+where ``aggregate_function`` is a name of an
+:ref:`aggregate function <aggregate-functions>`.
+
+``expression`` is a column reference, scalar function or literal.
```

Comment:
I think we should also document the first wildcard synopsis, e.g.:
"If a wildcard is used instead of an expression, no input value is given to the aggregation function a thus it may only useful (or even supported?) for the `count(*)` aggregation function."

Reply:
seems like it is `only` supported for the `count(*)`